store/ivy.xml

```diff
@@ -62,15 +62,15 @@
   <dependency org="commons-codec" name="commons-codec" rev="1.7"/>
   <dependency org="commons-io" name="commons-io" rev="1.4"/>
   <dependency org="commons-lang" name="commons-lang" rev="2.6"/>
-  <dependency org="commons-fileupload" name="commons-fileupload" rev="1.2.2"/>
+  <dependency org="commons-fileupload" name="commons-fileupload" rev="1.4"/>
```

Comment:
commons fileupload version is updated to 1.4 in feature/third-party-upgrade branch

Reply:
Fixed